facia/public/humans.txt

```diff
@@ -195,3 +195,8 @@ Location: London, UK
 
 Developer: Jamie Byers
 Location: London, UK
+
```

Comment:
I wasn't really sure whether *team* was the original team and *Alumni and Colleagues* was everyone else. Maybe I can be counted as *team* 🎈